packages/logos/azure-pipelines.yml

```diff
@@ -0,0 +1,21 @@
+# Node.js
```

Comment:
Hi Cassandra, can you please check the `diff` for this file and https://github.com/wfp/ui/blob/feat/monorepo-1.19.5/packages/logos/azure-pipelines.yml. Maybe this file is autogenerated by Azure Devops?

Reply:
I'm not sure if I get you but during the pipeline creation, it auto generated the azure-pipelines.yml file to copy and paste.

Reply:
No problem! :)